crowbar_framework/app/models/backup.rb

```diff
@@ -159,14 +159,19 @@ def save_archive
       return false
     end
 
+    if data.join("meta.yml").exist?
+      meta = YAML.load_file(data.join("meta.yml"))
```

Comment:
> Why wouldn't it be valid yaml when it is created by Crowbar?

How do you know the tarball is created by Crowbar and not hand-crafted by the user?